.github/workflows/ci.yml

```diff
@@ -45,6 +45,52 @@ jobs:
         run: |
           make -C build/ help | grep demo | tr -d '. ' | xargs make -C build/
           make -C demos/jobs/jobs_demo_mosquitto
+  build-check-demos-cpp:
+    runs-on: ubuntu-18.04
```

Comment:
We probably should use `ubuntu-latest` wherever we can unless there is a reason of using an older version

Reply:
I copied this from the other checks which use 18.04 due to compiler warnings in the coreHTTP library from gcc versions >= 10. Since this test doesn't use the -Werror flag, it's probably fine to use the latest here though